### PR TITLE
Odd Divisor Challenge

### PR DESCRIPTION
**Pull request by benjaminpearson**
Jay posted this challenge. Someone can verify I’ve understood the logic correctly in the simple test in there. We should add more values to test.

### CODE REVIEW

**Comment by benjaminpearson** [comment]
You can grep the test by running `npm test -- --grep=odd-divisor`

**Comment by benjaminpearson** [comment]
For Watch mode run `npm run test:watch -- --grep=odd-divisor`

File: challenges/odd-divisor/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+n=7
+f(1) + f(2) + f(3) + f(4) + f(5) + f(6) + f(7) = 1 + 1 + 3 + 1 + 5 + 3 + 7 = 21
+input: 1 ≤ n ≤ 1000000000, or even larger    
+
+The fastest function wins. Good luck!
+
+## Requirements
+
+- Fastest performing solution wins
+- Must work in browser and node.js
```
Comment (jaypeng2015) [inline challenges/odd-divisor/README.md]: Maybe remove this...
Any language should be allowed for this challenge.

**Review by jaypeng2015 — COMMENTED** [review]
Well, not sure if it's a good idea or not to use the repo because I was trying to say that javascript is not designed for this...

File: challenges/odd-divisor/README.md (hunk 1 above)
Comment (jaypeng2015) [reply challenges/odd-divisor/README.md]: Actually not.
Javascript is not good enough for this, I will explain that later in the lunch n learn.

**Pull request merged**